src/client/pages/Thema/Varen/VarenList.tsx

```diff
@@ -22,13 +26,31 @@ export function VarenList() {
   } = useVarenThemaData();
   useHTMLDocumentTitle(routeConfig.listPage);
 
-  const { title, displayProps, listPageRoute, filter, sort } =
-    tableConfig[kind];
-  const zaken = varenZaken.filter(filter).sort(sort);
+  const { title, displayProps, listPageRoute } = tableConfig[kind];
```

Comment:
Je zou evt nog een check kunnen doen op het bestaan van `kind in tableConfig` maar.. is dat nodig?